<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlightDetailDialog</class>
 <widget class="QDialog" name="FlightDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>航班详情</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #f8f9fa, stop:1 #e9ecef);
    font-family: &quot;Microsoft YaHei&quot;, &quot;PingFang SC&quot;, sans-serif;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="headerFrame">
     <property name="styleSheet">
      <string notr="true">QFrame {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #1e88e5, stop:1 #1565c0);
    border-radius: 10px;
    padding: 15px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="flightTitleLabel">
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: white;
    font-size: 22px;
    font-weight: bold;
    padding: 5px 0px;
}</string>
        </property>
        <property name="text">
         <string>✈️ 航班详情</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="flightNumberLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: white;
    font-size: 18px;
    font-weight: bold;
}</string>
          </property>
          <property name="text">
           <string>CA1234</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="airlineLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: rgba(255,255,255,0.9);
    font-size: 14px;
    background: rgba(255,255,255,0.2);
    padding: 4px 10px;
    border-radius: 15px;
}</string>
          </property>
          <property name="text">
           <string>中国国航</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="aircraftTypeLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: rgba(255,255,255,0.9);
    font-size: 12px;
    background: rgba(255,255,255,0.1);
    padding: 3px 8px;
    border-radius: 12px;
}</string>
          </property>
          <property name="text">
           <string>A320</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="routeFrame">
     <property name="styleSheet">
      <string notr="true">QFrame {
    background: white;
    border: 2px solid #e3f2fd;
    border-radius: 10px;
    padding: 20px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="departureLayout">
        <item>
         <widget class="QLabel" name="departureCityLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: #1e88e5;
    font-size: 20px;
    font-weight: bold;
}</string>
          </property>
          <property name="text">
           <string>北京</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="departureTimeLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: #333;
    font-size: 18px;
}</string>
          </property>
          <property name="text">
           <string>08:30</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="departureAirportLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: #666;
    font-size: 14px;
}</string>
          </property>
          <property name="text">
           <string>首都国际机场 T3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="arrowLayout">
        <item>
         <widget class="QLabel" name="durationLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: #666;
    font-size: 14px;
}</string>
          </property>
          <property name="text">
           <string>2h30m</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="arrowLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: #1e88e5;
    font-size: 28px;
    font-weight: bold;
}</string>
          </property>
          <property name="text">
           <string>→</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="arrivalLayout">
        <item>
         <widget class="QLabel" name="arrivalCityLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: #1e88e5;
    font-size: 20px;
    font-weight: bold;
}</string>
          </property>
          <property name="text">
           <string>上海</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="arrivalTimeLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: #333;
    font-size: 18px;
}</string>
          </property>
          <property name="text">
           <string>11:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="arrivalAirportLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: #666;
    font-size: 14px;
}</string>
          </property>
          <property name="text">
           <string>浦东国际机场 T2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane {
    border: 1px solid #e0e0e0;
    border-radius: 8px;
    background: white;
    margin-top: -1px;
}

QTabWidget::tab-bar {
    alignment: center;
}

QTabBar::tab {
    background: #f5f5f5;
    border: 1px solid #e0e0e0;
    border-bottom: none;
    padding: 8px 16px;
    margin-right: 2px;
    border-top-left-radius: 6px;
    border-top-right-radius: 6px;
    color: #666;
}

QTabBar::tab:selected {
    background: white;
    color: #1e88e5;
    font-weight: bold;
    border-bottom: 2px solid #1e88e5;
}

QTabBar::tab:hover {
    background: #e3f2fd;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="cabinTab">
      <attribute name="title">
       <string>舱位选择</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="cabinTable">
         <property name="styleSheet">
          <string notr="true">QTableWidget {
    background: white;
    border: none;
    gridline-color: #e0e0e0;
    alternate-background-color: #f8f9fa;
}

QTableWidget::item {
    padding: 12px 8px;
    border-bottom: 1px solid #f0f0f0;
}

QTableWidget::item:selected {
    background-color: #e3f2fd;
    color: #1e88e5;
}

QHeaderView::section {
    background: #1e88e5;
    color: white;
    padding: 10px;
    border: none;
    font-weight: bold;
}</string>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rowCount">
          <number>1</number>
         </property>
         <property name="columnCount">
          <number>5</number>
         </property>
         <row/>
         <column>
          <property name="text">
           <string>舱位类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>价格</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>行李额度</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>可用座位</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>操作</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="infoTab">
      <attribute name="title">
       <string>航班信息</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextEdit" name="flightInfoText">
         <property name="styleSheet">
          <string notr="true">QTextEdit {
    background: white;
    border: 1px solid #e0e0e0;
    border-radius: 8px;
    padding: 15px;
    font-size: 14px;
    color: #555;
}</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="footerFrame">
     <property name="styleSheet">
      <string notr="true">QFrame {
    background: white;
    border-top: 1px solid #e0e0e0;
    padding: 15px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background: #f5f5f5;
    color: #666;
    border: 1px solid #ddd;
    border-radius: 8px;
    padding: 10px 25px;
    font-size: 14px;
    font-weight: bold;
    min-width: 100px;
}

QPushButton:hover {
    background: #e0e0e0;
}

QPushButton:pressed {
    background: #d0d0d0;
}</string>
        </property>
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
